docs/io/configuration/components/spectrum.rst

```diff
@@ -15,7 +15,38 @@ just an integer. Finally, the method option selects the final spectral synthesis
 * real: construct spectrum from the real packet population alone
 * virtual: use the :ref:`virtual packet scheme <virtual_packets>` for spectral synthesis
 * integrated: use the :ref:`formal integral method <formal_integral>` of Lucy 1999
- 
+
```

Comment:
So this has nothing to do with the job you've done (which is great), and I wish I had noticed this prior. I don't believe that this info is up-to-date. @andrewfullard or @wkerzendorf can confirm, but I believe that the `method` argument does nothing. I tested this out and nothing changed (including in the hdf output). Currently, TARDIS can calculate all three types (as is described in the spectrum generation section of the docs). We just have extra settings that can be specified for the virtual and integrated methods.

Reply:
I did think it was weird that the method parameter was there when each method is called separately anyway. I presumed this served some function as setting a default method to be called. I will look into this

Reply:
For the time being, I will address this as a warning